src/solver/conv_ck_igemm_fwd_bias_res_add_activ_fused.cpp

```diff
@@ -72,7 +72,7 @@ namespace {
 
 struct CKArgs
 {
-    CKArgs(const ProblemDescription& problem)
+    CKArgs(const miopen::conv::ProblemDescription& problem)
```

Comment:
For reviewers:
https://github.com/ROCm/MIOpen/pull/2517/files#r1433171731

Reply:
@junliume FYI https://github.com/ROCm/MIOpen/pull/2517/files#r1433955991